### PR TITLE
Added BookingContainer component

### PR DESCRIPTION
**Pull request by robermancera**
# Description
We created the component of booking container, which will be able to reuse in other components.
 
 # Testing
Please let us know if we are doing something wrong, we also created a tiny component that we are about to use in the future ( - Fly Data -).
 
  ## Screenshots 
image

### CODE REVIEW

File: src/screens/BookingView/BookingView.jsx [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
 
 const BookingView = () => {
   return(
-    <View>
-      <Text>Booking</Text>
-    </View>
+    <BookingContainer
+      returnBtn={() => alert('return')}
+      next={() => alert('next')}
+    >
```
Comment (YilKanda) [inline src/screens/BookingView/BookingView.jsx:10]: You can use Alert.alert()

File: src/components/BookingContainer/BookingContainer.jsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        {children}
+      </View>
+      <View style={styles.buttonContainer}>
+        <CustomButton text='Next' onPress={next}/>
+      </View>
+    </View>
+  ); 
+};
+
+export default BookingContainer;
```
Comment (YilKanda) [inline src/components/BookingContainer/BookingContainer.jsx:31]: Add a blank line.

File: src/components/BookingContainer/BookingContainerStyle.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    flex: 1,
+    alignSelf: 'flex-start',
+    marginTop: 30,
+  },
+  flyData: {
+    backgroundColor: colors.lightGray,
+    padding: 35,
+    marginTop: 10,
+  },
+});
```
Comment (YilKanda) [inline src/components/BookingContainer/BookingContainerStyle.js:37]: Add a blank line.

File: src/components/TextBoxWithButton/TextBoxWithButtonStyle.js [hunk 4]
```diff
@@ -19,3 +19,4 @@ export const styles = StyleSheet.create({
     color: '#4d4d4d',
   },
 });
+
```
Comment (YilKanda) [inline src/components/TextBoxWithButton/TextBoxWithButtonStyle.js:22]: Remove the remaining blank line.

File: src/screens/BookingView/BookingView.jsx (hunk 1 above)
Comment (Shadic78) [reply src/screens/BookingView/BookingView.jsx]: Done

**Review by 96vladi — DISMISSED** [review]

**Review by YilKanda — DISMISSED** [review]
LGTM!

**Review by EusebioAjas — APPROVED** [review]

**Pull request merged**